crates/docs/src/lib.rs

```diff
@@ -101,23 +102,24 @@ pub fn generate_docs_html(root_file: PathBuf) {
     // Insert asset urls & sidebar links
     let template_html = assets
         .raw_template_html
-        .replace("<!-- search.js -->", "/search.js")
-        .replace("<!-- styles.css -->", "/styles.css")
+        .replace("<!-- search.js -->", "search.js")
+        .replace("<!-- styles.css -->", "styles.css")
         .replace("<!-- favicon.svg -->", "/favicon.svg")
         .replace(
             "<!-- Prefetch links -->",
             loaded_module
                 .docs_by_module
                 .iter()
                 .map(|(_, module)| {
-                    let href = module_link_url(module.name.as_str());
+                    let href = module.name.as_str();
```

Comment:
use a relative link instead

Reply:
Hm, I think these started out as relative and then got changed to root...but I don't remember why! I'll try to dig it up...

Reply:
@rtfeldman should we just merge this and see if any issues pop up? It's been working well for the basic-cli 0.3.2 docs

Reply:
Sure, that works 👍

Reply:
@Anton-4 ok turns out it breaks all the CSS if you refresh 😂

Reply:
fixed :)